linera-views/tests/store_tests.rs

```diff
@@ -75,17 +75,34 @@ async fn test_reads_rocks_db() {
 #[cfg(with_dynamodb)]
 #[tokio::test]
 async fn test_reads_dynamo_db() {
+    use linera_views::store::AdminKeyValueStore as _;
+
     for scenario in get_random_test_scenarios() {
         let store = linera_views::dynamo_db::DynamoDbStore::new_test_store()
             .await
             .unwrap();
+        let store = store.clone_with_root_key(&[]).unwrap();
         run_reads(store, scenario).await;
     }
 }
 
 #[cfg(with_scylladb)]
 #[tokio::test]
 async fn test_reads_scylla_db() {
+    use linera_views::store::AdminKeyValueStore as _;
+
+    for scenario in get_random_test_scenarios() {
+        let store = linera_views::scylla_db::ScyllaDbStore::new_test_store()
+            .await
+            .unwrap();
+        let store = store.clone_with_root_key(&[]).unwrap();
+        run_reads(store, scenario).await;
+    }
+}
+
+#[cfg(with_scylladb)]
+#[tokio::test]
+async fn test_reads_scylla_db_no_journaling() {
```

Comment:
Running locally those tests, it is impossible to see some differences in the timing. And running both tests takes the same runtime as one single test.

Also, I object to the `no_journaling` terminology:
* The journaling is still present in the journaling. If we are in exclusive access then operation fail.
* The difference is potentially about caching not journaling. And there is still caching, just caching of values, not absence of values.
* But actually there is no difference. The difference between exclusive access and no exclusive access is about the checking of absence, and that caching of absence is not going to be present.

Reply:
ok fair. let me rename to `test_reads_scylla_db_no_root_key`